pkg/ipmi/ipmi.go

```diff
@@ -32,31 +35,31 @@ const (
 	_IPMI_SYSTEM_INTERFACE_ADDR_TYPE = 0x0c
 
 	// IPM Device "Global" Commands
-	_BMC_GET_DEVICE_ID = 0x01
+	BMC_GET_DEVICE_ID = 0x01
 
 	// BMC Device and Messaging Commands
-	_BMC_SET_WATCHDOG_TIMER     = 0x24
-	_BMC_GET_WATCHDOG_TIMER     = 0x25
-	_BMC_SET_GLOBAL_ENABLES     = 0x2E
-	_BMC_GET_GLOBAL_ENABLES     = 0x2F
-	_SET_SYSTEM_INFO_PARAMETERS = 0x58
-	_BMC_ADD_SEL                = 0x44
+	BMC_SET_WATCHDOG_TIMER     = 0x24
+	BMC_GET_WATCHDOG_TIMER     = 0x25
+	BMC_SET_GLOBAL_ENABLES     = 0x2E
+	BMC_GET_GLOBAL_ENABLES     = 0x2F
+	SET_SYSTEM_INFO_PARAMETERS = 0x58
+	BMC_ADD_SEL                = 0x44
```

Comment:
Should Cmd in Msg get its own type? 

type Command uint16

BMC_ADD_SEL Command = 0x44

Would make it clear to people like me who don't know shit which constants go in the Netfn field and which in the command

Reply:
Added types for cmd and netfn